superset/tasks/cache.py

```diff
@@ -47,6 +48,23 @@ def get_payload(chart: Slice, dashboard: Optional[Dashboard] = None) -> dict[str
     return payload
 
 
+def is_secure_url(url: str) -> bool:
```

Comment:
would fit nicely in `superset/utils/urls.py`

Reply:
Thank you for the suggestion! I have updated the code and moved the function to `urls.py`